jina/peapods/pod.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, args: Union['argparse.Namespace', Dict]):
 
         :param args: arguments parsed from the CLI
         """
-        self.peas = []
+        self.peas: List[Union[RemotePea, ContainerPea, HeadPea, TailPea, BasePea]] = []
```

Comment:
They are all actually BasePea instances, so we can have List[BasePea]

Reply:
HIi @JoanFM ,

Thanks for your response, pushing a fixup commit :).

Reply:
Cool! Lint also complaints about some unknown types. Please make sure you add the needed import statements.

Reply:
I have resolved the issues.